### ISSUE TITLE
NTR: Infection-related

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5192]
"-Seizure
---Status epilepticus
---Reflex seizure
---Motor seizure
---Focal-onset seizure
---Generalized-onset seizure
---Dialeptic seizure
---Neonatal seizure
---NTR: Infection-related
-----Gastroenteritis-related afebrile seizure
-----Seizure precipitated by febrile infection
label: Infection-related
def: Seizure associated with a presumed or proven infection (excluding infection of the central nervous system) or inflammatory state without an alternative precipitant such as metabolic derangement, and regardless of the presence or absence of a fever.
comment: This includes seizures in the context of febrile or afebrile infection.
http://webprotege.stanford.edu/RiCnLhdRh6VgSt0KNJ92ib: Epilepsiome
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney"
PMID:28276060: https://www.ncbi.nlm.nih.gov/pubmed/PMID:28276060

**Comment by pnrobinson** [comment]
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney


Should the name of this term be Infection-related seizure?

Also, what should we do with "Symptomatic seizure"? 
https://hpo.jax.org/app/browse/term/HP:0011145

I am not sure this is a great term, but Infection related belongs here too and some physicians will expect to  see a term like this?

**Comment by DLewisSmith** [comment]
Yes, good catch Peter it should be ‘Infection-related seizure’ (although this label may eventually need reviewing with feedback from other experts as we have invented it to cover a concept of seizures related to an inflammatory or infective state but not necessarily the presence of a fever).

'Acute symptomatic seizure’ is one we should probably try to have a TC discussion about. It is a term that is in use and which has a recommended definition from an ILAE subcommission (https://www.ilae.org/files/dmfile/Recommendation-for-definition-of-acute-seizure-Beghi2010-Epilepsia.pdf). This recommendation does include ‘Febrile seizure’ without an age cut-off as a type of Acute symptomatic seizure, but some clinicians would consider this different because by definition 'Acute symptomatic seizures' do not contribute to a diagnosis of epilepsy, where as 'febrile seizures' extending beyond the 6th birthday are diagnostic of febrile seizure plus (an epilepsy syndrome).

The questions to consider include:
1) Does/should the HPO include distinct phenotypes based on specific aetiologies?
2) Does/should the HPO include non-pathological phenotypes?

An 'Acute symptomatic seizure' occurs because of an extrinsic insult to a brain rather than any intrinsic predisposition to seizures. Thus, one could argue that ‘Acute symptomatic seizure' could be likened to ‘Bleeding due to trauma’. Conversely, one might say that ‘Acute symptomatic seizure’ is an important phenotypic term for severe hypoparathyroidism (via hypocalcaemia), insulinoma (via hypoglycaemia), or alcoholism.

I agree that theoretically the term could be useful in the future: are there genetic factors that predispose to ‘Acute symptomatic seizures' due to a mild, usually asymptomatic hypocalcaemia/hypoglycaemia/traumatic brain injury? We don’t know. If included we could generate child terms for the different aetiologies and suggested cut-offs from the paper above. However, I think we would need to consider its relationship to 'febrile seizure’ and its close relatives carefully.

**Comment by pnrobinson** [comment]
1) Does/should the HPO include distinct phenotypes based on specific aetiologies?
A: Sometimes, if it is simply so important clinically that there is no other way (e.g., symptomatic seizure)
2) Does/should the HPO include non-pathological phenotypes?
A: If they are diagnostically important, yes.

I will add the term as recommended and we can discuss this in a second stage?

**Comment by pnrobinson** [comment]
added as HP_0032892,  ‘Infection-related seizure’.
I understand that we may want to review this topic, and I will leave the issue open so we can decide how/where/when to do so.

**Comment by pnrobinson** [comment]
@DLewisSmith  can this be closed?

**Comment by DLewisSmith** [comment]
@pnrobinson  Yes I think so.

**Comment by pnrobinson** [comment]
closing
